extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -59,10 +60,10 @@ describe('ProjectsController: project controller operations', function (): void
 			testContext.apiWrapper.setup(x => x.showErrorMessage(TypeMoq.It.isAny())).returns((s) => { console.log('we throwin'); throw new Error(s); });
 
 			const projController = new ProjectsController(testContext.apiWrapper.object, new SqlDatabaseProjectTreeViewProvider());
-			const project = new Project('FakePath');
+			const project = new Project('FakePath', new ApiWrapper());
```

Comment:
Instead of `new ApiWrapper()`, this should be `testContext.apiWrapper.object`, which gives us the mocked version for tests instead of the real one.

Reply:
Though I suppose it's really just going to stay without ApiWrapper entirely just 'cause of the changes in that file :)